src/components/collapsible/Collapsible.js

```diff
@@ -75,7 +97,7 @@ class Collapsible extends React.Component<Props, State> {
     render() {
         const { isOpen }: State = this.state;
         const {
-            animationDuration,
+            animationDuration = 100,
```

Comment:
Already defaulted in defaultProps above

Reply:
I did this to make flow happy. Flow doesn't seem to understand the `static defaultProps`.

Therefore, it warned that I could get `transaction-duration: undefinedms`.